app/views/admins/admins/edit_password.html.slim

```diff
@@ -0,0 +1,38 @@
+.container.p-0
+  .card
+    .card-header
+      h5.card-title.mb-0
+        = t ('admins.passwords.edit.change_password_header')
+    .card-body
+      form
+      = form_for @admin, url: { action: 'update_password' } do |form|
+        .row
+          .col-md-12
+              .form-group
+              label
+                = t ('admins.passwords.edit.form.current_password_label')
+              = form.password_field :current_password,
+              class: 'form-control'
+              ol.mb-0.mt-1
+                - @admin.errors[:current_password].each do |msg|
```

Comment:
consider [THIS WAY](https://apidock.com/rails/ActionView/Helpers/ActiveRecordHelper/error_messages_for)

Reply:
Tried to use `error_messages_for`, but it wasn`t working, so I haven`t changed that.

Reply:
why it doesn't work?

Reply:
in docs it is said that method is deprecated
![Screenshot from 2021-12-09 16-51-25](https://user-images.githubusercontent.com/26040954/145419327-b1cfabac-a3d5-444f-9cb2-59b2b810aa22.png)
d